gatsby-node.js

```diff
@@ -14,7 +14,9 @@ exports.createPages = ({ graphql, actions }) => {
         `
           {
             allMarkdownRemark(
-              filter: { fields: { slug: { nin: ["", "homepage"] } } }
+              filter: {
+                fields: { slug: { nin: ["community-page", "homepage"] } }
```

Comment:
```suggestion
                fields: { slug: { nin: ["", "community-page", "homepage"] } }

```

Ignore the template file otherwise `/learn` will have a weird sidebar.